packages/fxa-auth-server/scripts/check-users.js

```diff
@@ -0,0 +1,217 @@
+        verifyHash
+      );
+
+      // Check to see if user has MFA enabled
+      let mfaEnabled = false;
+      try {
+        const totpToken = await this.db.totpToken(accountRecord.uid);
+        if (totpToken) {
+          mfaEnabled = true;
+        }
+      } catch (err) {}
+
+      const s = {
+        email: this.email,
+        exists: true,
+        passwordMatch,
+        mfaEnabled,
+        keysChangedAt: accountRecord.keysChangedAt,
+        profileChangedAt: accountRecord.profileChangedAt,
+        hasSecondaryEmails: accountRecord.emails.length > 1,
+        isPrimaryEmailVerified: accountRecord.primaryEmail.isVerified,
+      };
+      const stat = `${s.email},${s.exists},${sanitizeValue(
+        s.passwordMatch
+      )},${sanitizeValue(s.mfaEnabled)},${sanitizeValue(
+        s.keysChangedAt
+      )},${sanitizeValue(s.profileChangedAt)},${sanitizeValue(
+        s.hasSecondaryEmails
+      )},${sanitizeValue(s.isPrimaryEmailVerified)}`;
+
+      console.log(stat);
+
+      return s;
+    } catch (err) {
+      return {
+        exists: false,
+        email: this.email,
+      };
+    }
+  }
+}
+
+class CheckUsers {
+  constructor(filename) {
+    this.users = [];
+    this.db = undefined;
+    this.filename = filename;
+  }
+
+  getItems() {
+    try {
+      const input = fs
+        .readFileSync(path.resolve(this.filename))
+        .toString('utf8');
+
+      if (!input.length) {
+        return [];
+      }
+
+      // Parse the input file CSV style
+      return input.split(/\n/).map((s) => {
+        const delimiter = program.delimiter || ':';
+        const email = s.substring(0, s.indexOf(delimiter));
+        const password = s.substring(s.indexOf(delimiter) + 1, s.length);
+        return new User(email, password, this.db);
+      });
+    } catch (err) {
+      console.error('No such file or directory');
+      process.exit(1);
+    }
+  }
+
+  async load() {
+    this.db = await AuthDB.connect(config);
+    this.users = this.getItems();
+    console.info(
+      '%s accounts loaded from %s',
+      this.users.length,
+      this.filename
+    );
+  }
+
+  async close() {
+    await this.db.close();
+  }
+
+  async userStats() {
+    const stats = [];
+    for (const user of this.users) {
+      stats.push(await user.stats());
+    }
+    this.stats = stats;
+  }
+
+  saveStats() {
+    const stats = this.stats;
+    const output = [
+      'email,exists,passwordMatch,mfaEnabled,keysChangedAt,profileChangedAt,hasSecondaryEmails,isPrimaryEmailVerified',
+    ];
+
+    output.push(
+      ...stats.map((s) => {
+        return `${s.email},${s.exists},${sanitizeValue(
+          s.passwordMatch
+        )},${sanitizeValue(s.mfaEnabled)},${sanitizeValue(
+          s.keysChangedAt
+        )},${sanitizeValue(s.profileChangedAt)},${sanitizeValue(
+          s.hasSecondaryEmails
+        )},${sanitizeValue(s.isPrimaryEmailVerified)}`;
+      })
+    );
+    const outputFile = program.output || 'stats.csv';
```

Comment:
similar to my previous nit, you should be able to set this in the commander config on L24 like:
```diff
-.option('-o, --output <filename>', 'Output filename to save results to')
+.option('-o, --output <filename>', 'Output filename to save results to', 'stats.csv')
```